ddp-workspace/projects/ddp-mbc/src/app/components/about-us/about-us.component.ts

```diff
@@ -49,11 +49,9 @@ import { ToolkitConfigurationService } from 'toolkit';
               <a [href]="countMeInUrl" class="Link" target="_blank" translate>Toolkit.AboutUs.ContentLink</a>.
             </p>
             <p class="PageContent-text">
-              <span translate>Toolkit.AboutUs.Content.Paragraph2.Pt1</span>
-              <span class="Italic" translate>Toolkit.Common.Organization</span>'s
```

Comment:
Why did you skip `class="Italic"` ?

Reply:
I use <em> element in en.json file which makes it look the same

Reply:
Sorry, didn't notice it firstly

Reply:
Interesting, that Spanish  version is a bit different: 
> We leverage the power of <em>Count Me In</em>'s resources

vs 

> Usamos los recursos y tecnología genómica del Instituto Broad (Broad Institute)